integritybackend/config.py

```diff
@@ -65,8 +65,57 @@ def all_orgs(self):
         return self.config.keys()
 
     def get(self, org_id):
-        """Gets configuration dictionary for an org id."""
-        return self.config.get(org_id)
+        """Gets configuration dictionary for an org."""
+        if org_id in self.config:
+            return self.config.get(org_id)
+        else:
+            raise Exception(f"No organization with ID {org_id}")
+
+    def get_collections(self, org_id):
+        """Gets collection array for an org id."""
+        org_dict = self.get(org_id)
+        if "collections" in org_dict:
+            return org_dict.get("collections")
+        else:
+            raise Exception(f"No collections defined in {org_id}")
+
+    def get_collection(self, org_id, collection_id):
+        """Gets specific collection for an org."""
+        org_collections = self.get_collections(org_id)
+        collection_config = next(
+            (
+                c
+                for c in org_collections.values()
+                if c.get("conf").get("id") == collection_id
+            ),
+            None,
+        )
+        if collection_config is None:
+            raise Exception(f"No collection in {org_id} with ID {collection_id}")
+        return collection_config.get("conf")
```

Comment:
Can you paste the return value of this? As a client, calling `get_collection` it's not clear what I will get. If it's a schema that is not the same as the raw json file, for the collection, that's even more weird.

Reply:
This should now be clearer as i am using the original JSON schema without `conf`